packages/shared/lib/services/sync/run.service.ts

```diff
@@ -325,7 +331,12 @@ export default class SyncRun {
                     const message = `The integration was run but there was a problem in retrieving the results from the script "${this.syncName}"${
                         syncData?.version ? ` version: ${syncData.version}` : ''
                     }`;
-                    await this.reportFailureForResults(message);
+
+                    if (error.type === 'script_cancelled') {
+                        await this.reportFailureForResults(error.message);
+                    } else {
+                        await this.reportFailureForResults(message);
```

Comment:
could this be more generic: `await this.reportFailureForResults(error?.message || message);`

Reply:
I don't want to change the behavior here too much. I believe a NangoError will have a message but we don't necessarily want to expose that to the user. Error messaging should be cleaned up but I think that is out of scope of this work.